### PR TITLE
enh: populate loadGenerator in SMP results

### PR DESCRIPTION
**Pull request by kushthedude**

### CODE REVIEW

**Comment by kushthedude** [comment]
The complete rewrite of model-test file was due to the `gofmt -s -w ./`

File: models/load-test.go [hunk 1]
```diff
@@ -128,20 +128,20 @@ func (m *MesheryResult) ConvertToSpec() (*PerformanceSpec, error) {
...
-	// retcodes[200] = 10
 	m.Result["RetCodes"] = retcodes
+	loadGenerator := m.Result["load-generator"].(string)
 	logrus.Debugf("result to be converted: %+v", m)
 	if m.Result["RunType"].(string) == "HTTP" {
 		httpResults := &fhttp.HTTPRunnerResults{}
 		resJ, err := json.Marshal(m.Result)
 		if err != nil {
-			err = errors.Wrap(err, "unable while converting Meshery result to Benchmark Spec")
+			err = errors.Wrap(err, "unable while converting meshery result to benchmark spec")
```
Comment (leecalcote) [inline models/load-test.go:138]:
```suggestion
			err = errors.Wrap(err, "unable to marshal Meshery performance results into SMP")
```

File: models/load-test.go [hunk 2]
```diff
@@ -128,20 +128,20 @@ func (m *MesheryResult) ConvertToSpec() (*PerformanceSpec, error) {
...
 			logrus.Error(err)
 			return nil, err
 		}
 		err = json.Unmarshal(resJ, httpResults)
 		if err != nil {
-			err = errors.Wrap(err, "unable while converting Meshery result to Benchmark Spec")
+			err = errors.Wrap(err, "unable while converting meshery result to benchmark spec")
```
Comment (leecalcote) [inline models/load-test.go:144]:
```suggestion
			err = errors.Wrap(err, "could not extract Meshery performance results from SMP")
```

**Review by leecalcote — APPROVED** [review]
LGTM

**Pull request merged**